worlds/hk/__init__.py

```diff
@@ -603,15 +640,15 @@ def get_filler_item_name(self) -> str:
                     'RandomizeGeoRocks', 'RandomizeSoulTotems', 'RandomizeLoreTablets', 'RandomizeJunkPitChests',
                     'RandomizeRancidEggs'
             ):
-                if getattr(self.multiworld, group):
+                if getattr(self.options, group):
```

Comment:
How is the truthiness of the options classes handled? Might need a `.value` on this

Reply:
which ones are you worried about? because options are pretty good at truthy handling, 
i can always cast to a bool if you want it to be explicit too

Reply:
FYI confirmed that a yaml with all the relevant filler shuffle options turned off and with 9 shop slots for each option 
had (except the one Collector's map and Salubra's blessing) only "one_geo" and "soul_refill" filler items 

strange unrelated note: i had to filter down to items with .code not None because for some reason lifeblood cocoons were being created as events and i'm not sure why